src/Category/Applicative/Indexed.agda

```diff
@@ -94,9 +94,10 @@ record RawIAlternative
 -- Applicative functor morphisms, specialised to propositional
 -- equality.
 
-record Morphism {I : Set i} {F₁ F₂ : IFun I f}
-                (A₁ : RawIApplicative F₁)
-                (A₂ : RawIApplicative F₂) : Set (i ⊔ suc f) where
+record ApplicativeMorphism
```

Comment:
Can we keep these names & simply hide the `Morphism` import?

Reply:
Yes, you both know much better the intricacies of the `agda-stdlib` but please note that @MatthewDaggitt was [asking explicitly](https://github.com/agda/agda-stdlib/issues/1206#issuecomment-631807844) for the import.

I can change it to whatever you agree.

Reply:
Re-exporting the definition from various modules is different from
hiding it when importing `Function.Core` inside of the `Category.*` modules.

I don't think it's worth introducing breaking changes for a simple type alias.

Reply:
I was hiding `Morphism` when importing `Function` in some modules, Travis is happy now :relieved: